app/frontend/src/views/NotFound.vue

```diff
@@ -1,17 +1,23 @@
 <template>
   <v-container class="text-center">
-    <h1 class="my-6">404: Page not found. :(</h1>
+    <h1 class="my-6" :lang="lang">{{ $t('trans.notFound.pageNotFound') }}</h1>
```

Comment:
Does this also need ` :class="{ 'dir-rtl': isRTL }"`?

Reply:
Yes